src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,48 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
+
+import { UserInfo } from '../UserInfo';
+
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+function findUser(userId: number, users: User[]): User {
```

Comment:
The function findUser uses a potentially expensive operation (Array.prototype.find) every time a user needs to be found. This could be optimized by transforming usersFromServer into an object where user IDs are keys, which would make lookups more efficient.